src/actor.js

```diff
@@ -195,7 +195,7 @@ export const getEnv = () => {
  *
  * If the user function returns a promise, it is considered asynchronous:
  * ```javascript
- * const request = require('request-promise-native');
+ * const { requestAsBrowser } = require('some-request-library');
```

Comment:
Wouldn't it be better to have all these examples working? I.e. to really import requestAsBrowser from Apify or request package.

Reply:
I don't know, they'd have to look like this:

```javascript
const Apify = require('apify')' 
 
Apify.main(() => {
    // My asynchronous function that returns a promise
    return Apify.utils.requestAsBrowser('http://www.example.com').then(({ body }) => {
        console.log(body);
    });
});
```